jenkins/rpm/v2/cortx-ha.spec

```diff
@@ -39,28 +39,18 @@ exit 0
 
 %post
 HA_DIR=<HA_PATH>/ha
-RES_AGENT="/usr/lib/ocf/resource.d/seagate"
```

Comment:
Can we copy and modify all these files in jenkins/rpm/v2/k8s and copy the file to appropriate place in build.sh before build/setup starts.

Reply:
1. Original files are intact
2. New files with modified contents are moved to jenkins/rpm/v2/k8s
3. Modified build.sh to use this new path